content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,564 @@
+of vendor- and community-built Prometheus exporters to aggregate and import data 
+into Prometheus.
+
+While you can use Prometheus to monitor a variety of infrastructure and 
+application metrics, one of its most popular use cases is to monitor Kubernetes. 
+This is the aspect of Prometheus monitoring that we will focus on in this 
+article.
+
+## Prometheus Metrics with OpenTelemetry
+In this section, you will learn about a couple of OTel collector components that 
+demonstrate the interoperability between OTel and Prometheus.  
+
+First, let’s do a quick refresher on the [collector](https://opentelemetry.io/docs/collector/) 
+– it’s an OTel component that can be used to collect and export data to and from 
+multiple sources and destinations. It also handles additional telemetry 
+processing, such as modifying data attributes and scrubbing personally 
+identifiable information. For example, you can use Prometheus SDKs to generate 
+metrics, ingest them with the collector, do some processing (if desired) and 
+then forward them to your chosen backend. 
+
+![Diagram showing the different components of the OTel collector](OTel-collector-refresher.png)
+
+The [Prometheus receiver](https://opentelemetry.io/docs/kubernetes/collector/components/#prometheus-receiver) allows you to collect metrics from any software that 
+exposes Prometheus metrics. It serves as a drop-in replacement for Prometheus to 
+scrape your services, and supports the [full set](https://github.com/prometheus/prometheus/blob/v2.28.1/docs/configuration/configuration.md#scrape_config) of configurations in 
+`scrape_config`. 
+
+If you are interested in [exemplars](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exemplars), which is a recorded value that associates OTel context to a metric 
+event, you can also use the Prometheus receiver to ingest them in the Prometheus 
+format and convert it to OTLP format. This enables you to correlate traces with 
+metrics. 
+
+Something to consider with this component is that it is under active 
+development; as such, it has several [limitations](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md#%EF%B8%8F-warning), including that it’s a stateful component. Additionally, it is recommended to not 
+use this component when multiple replicas of the collector are run, because in 
+this state: 
+* The collector is unable to auto-scale the scraping 
+* If the replicas are running with the same config, it will scrape the targets 
+multiple times
+* You will need to configure each replica with a different scraping config if 
+you want to manually shard the scraping
+
+The [Prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter#prometheus-exporter) allows you to ship data in the 
```

Comment:
Thank you @gouthamve ! I modified this section like so:

> For exporting metrics from the OTel Collector to Prometheus, you have two options: 
> the [Prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter#prometheus-exporter), and the [Prometheus Remote Write exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/README.md).
> 
> The Prometheus exporter allows you to ship data in the Prometheus format, which is 
> then scraped by a Prometheus server. It's used to report metrics via the Prometheus 
> scrape HTTP endpoint. You can learn more by trying out this [example](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/prometheus). However, the scraping won't really scale, as all the metrics are sent in a single scrape. 
> 
> To get around the scaling concern, you can alternatively use the Prometheus Remote Write exporter, which allows you to push data to Prometheus from multiple Collector 
> instances with no issues. Since Prometheus also accepts remote write ingestion, you 
> can also use this exporter if you are generating OTel metrics and want to 
> ship them to a backend that is compatible with Prometheus remote write. Learn more about the architecture of both exporters [here](https://grafana.com/blog/2023/07/20/a-practical-guide-to-data-collection-with-opentelemetry-and-prometheus/#6-use-prometheus-remote-write-exporter). 

Is this technically accurate, and does it effectively convey your recommendation?

Reply:
This is great!